### PR TITLE
Notification Service Endpoints

### PR DESCRIPTION
**Pull request by Whomstt**
Endpoint for sending notifications at "notifications/send".
Attach "recipient" email and "type" as JSON
Current existing types: (account_creation or game_update)

### CODE REVIEW

File: notification-service/src/main/java/com/lms/notificationservice/NotificationServiceApplication.java [hunk 1]
```diff
@@ -4,24 +4,36 @@
...
+        Dotenv dotenv = Dotenv.configure().directory("../").load();
+        System.setProperty("NOTIFICATION_SERVICE_APP_PASSWORD", dotenv.get("NOTIFICATION_SERVICE_APP_PASSWORD"));
+        SpringApplication.run(NotificationServiceApplication.class, args);
+    }
+
+    @EventListener(ApplicationReadyEvent.class)
+    public void onApplicationReady() {
+        System.out.println("Notification Service is up and running and ready to accept requests.");
+
+        // Send a test notification to the email address below
```
[the remark this answers is not in the document]
Comment (Whomstt) [reply notification-service/src/main/java/com/lms/notificationservice/NotificationServiceApplication.java:33]: This was just to test the service before api calls were implemented, will remove

File: .github/workflows/main_lastman-standing-app.yml [hunk 2]
```diff
@@ -22,33 +25,35 @@ jobs:
           java-version: '17'
           distribution: 'microsoft'
 
-      - name: Build with Maven
-        run: mvn clean install
+      - name: unit tests
```
Comment (olanhealy) [inline .github/workflows/main_lastman-standing-app.yml:28]: Im not sure what this is? I thought mvn clean install ran unit tests fine but could be wrong

**Review by olanhealy — CHANGES_REQUESTED** [review]

File: .github/workflows/main_lastman-standing-app.yml (hunk 2 above)
Comment (olanhealy) [reply .github/workflows/main_lastman-standing-app.yml:28]: Is this branch also not setup for pipeline yet?

Comment (Whomstt) [reply .github/workflows/main_lastman-standing-app.yml:28]: This is already in main, I just merged main into my branch to solve conflicts.

Comment (olanhealy) [reply .github/workflows/main_lastman-standing-app.yml:28]: Ah okay sound, good work will approve, @CaoimhinHawk1 need pipeline to run on this too in future

**Review by olanhealy — APPROVED** [review]
Approving as comments clarified. very good work well done

**Review by CaoimheCahill — APPROVED** [review]

**Pull request merged**